ci-operator/step-registry/osd/gather-extra/osd-gather-extra-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
+AWS_ACCOUNT_ID=$(cat /var/run/hcm-jobs-logs-s3-bucket-storage/account_id)
+HIVE_LOGS_BUCKET=$(cat /var/run/hcm-jobs-logs-s3-bucket-storage/bucket)
+CLUSTER_NAME=$(cat "${SHARED_DIR}/cluster-name")
+CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
+ROLE_ARN=arn:aws:iam::${AWS_ACCOUNT_ID}:role/ocp-trt-nightly-hive-logs-read-only
+ROLE_SESSION_NAME=OSDGatherExtraInstallLogs
+
+# shellcheck disable=SC2183,SC2046
+export $(printf "AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s AWS_SESSION_TOKEN=%s" \
+	$(aws sts assume-role \
+		--role-arn "${ROLE_ARN}" \
+		--role-session-name "${ROLE_SESSION_NAME}" \
+		--query "Credentials.[AccessKeyId,SecretAccessKey,SessionToken]" \
+		--output text))
+
+# copy from hive install bucket into artifacts dir
+aws s3 cp --recursive \
+	s3://"${HIVE_LOGS_BUCKET}"/"${CLUSTER_NAME}"-uhc-staging-"${CLUSTER_ID}" \
+	"${ARTIFACT_DIR}"/
```

Comment:
`ARTIFACT_DIR` is a public folder, is there any confidential message in hive log? If there is, you should use SHARED_DIR instead.